### DISCUSSION TITLE
How do I disable MPV showing up in the Windows lock screen?

### DISCUSSION THREAD

**Discussion by Danfun64** [mpv-player/mpv#14870] [Q&A] [answered]
I don't like videos being accessible from the windows lock screen. I know Chrome has a similar issue, but that can be turned off with the setting "Hardware Media Key Handling". What is the equivalent for Windows?

I am running Windows 11 with mpv-x86_64-20240915-git-ef19a4a.

![20240916_143716](https://github.com/user-attachments/assets/7f778f2a-273c-4b21-a85f-298374817967)

**Comment by kasper93** [comment]
Duplicate of: https://github.com/mpv-player/mpv/issues/14857 and https://github.com/mpv-player/mpv/discussions/14856

**Comment by Danfun64** [comment]
Is there a way to set it to default at least, like in a configuration file or something? I'd like to be able to open videos without using the command prompt.

**Reply by kasper93** [reply, marked as answer]
https://mpv.io/manual/master/#configuration-files
https://mpv.io/manual/master/#files
https://mpv.io/manual/master/#files-on-windows

**Reply by Danfun64** [reply]
creating a mpv.conf in C:\users\USERNAME\AppData\Roaming\mpv\mpv.conf did the trick. Thanks.
